var/spack/repos/builtin/packages/fseq/package.py

```diff
@@ -0,0 +1,41 @@
+##############################################################################
+# Copyright (c) 2013-2016, Lawrence Livermore National Security, LLC.
+# Produced at the Lawrence Livermore National Laboratory.
+#
+# This file is part of Spack.
+# Created by Todd Gamblin, tgamblin@llnl.gov, All rights reserved.
+# LLNL-CODE-647188
+#
+# For details, see https://github.com/llnl/spack
+# Please also see the NOTICE and LICENSE files for our notice and the LGPL.
+#
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU Lesser General Public License (as
+# published by the Free Software Foundation) version 2.1, February 1999.
+#
+# This program is distributed in the hope that it will be useful, but
+# WITHOUT ANY WARRANTY; without even the IMPLIED WARRANTY OF
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the terms and
+# conditions of the GNU Lesser General Public License for more details.
+#
+# You should have received a copy of the GNU Lesser General Public
+# License along with this program; if not, write to the Free Software
+# Foundation, Inc., 59 Temple Place, Suite 330, Boston, MA 02111-1307 USA
+##############################################################################
+from spack import *
+
+
+class Fseq(Package):
+    """F-Seq: A Feature Density Estimator for High-Throughput Sequence Tags"""
+
+    homepage = "http://fureylab.web.unc.edu/software/fseq/"
+    url      = "http://fureylab.med.unc.edu/fseq/fseq_1.84.tgz"
+
+    version('1.84', 'f9124ad0f45c60f3a7eb74dde8c945b9')
+
+    depends_on('jdk')
```

Comment:
This should probably be `type=('build', 'run')` unless it actually links to the jdk libraries.